google/cloud/spanner/internal/connection_impl.cc

```diff
@@ -830,6 +841,7 @@ ConnectionImpl::PartitionQueryImpl(
         Idempotency::kIdempotent,
         [&stub](grpc::ClientContext& context,
                 google::spanner::v1::PartitionQueryRequest const& request) {
+          RouteToLeader(context);
```

Comment:
Why no `if (route_to_leader)` here?

Reply:
Because `PartitionQuery()` calls should be unconditionally routed to the leader.

Reply:
Consider a comment (here and the other places where it is not conditional).  You probably won't forget that, I will.

Reply:
Done.